src/main/java/com/example/parser/utils/Parser.java

```diff
@@ -55,22 +48,8 @@ public void parser(String filePath) throws IOException, ParseException {
             JsonList.add(permit);
             fileCounter++;
         }
+        parsingResponse.setFileRecordsCounter(fileCounter);
 
-        Set<PermitForEmissionsOfPollutants> union = new HashSet(JsonList);
-        union.removeAll(BD_list);
-
-        for (PermitForEmissionsOfPollutants k : union) {
-            permitRepository.save(k);
-            dataBaseCounter++;
-        }
-
-        long end = System.currentTimeMillis();
-        time = TimeUnit.MILLISECONDS.toSeconds(end - start);
-        date = new Date();
-
-        log.info("1) Time spent processing the file {} seconds", time);
-        log.info("2) Date the file was processed {}", date);
-        log.info("3) How many records were read from the file {}", fileCounter);
-        log.info("4) How many new records have been added to the database {}", dataBaseCounter);
+        return JsonList;
```

Comment:
This variable name must start from lower case symbol.

Reply:
done